src/pages/index.js

```diff
@@ -116,9 +117,18 @@ export default () => {
           />
         </div>
       </Banner>
-      <div id='about' style={{ minHeight: '100vh' }}>
+      <div id='about'>
         <About />
+        <div
+          id='why-mun'
```

Comment:
What is the need of assigning this id ??

Reply:
removed this.